src/DiagramGenerator/Diagram/Board.php

```diff
@@ -216,7 +209,7 @@ public function drawBorder()
      */
     public function draw()
     {
-        $this->image->setImageFormat('png');
+        $this->image->setImageFormat(Texture::IMAGE_FORMAT_PNG);
```

Comment:
yes this is wrong, but changing that is scheduled for the next PR, along with adding `pngquant`
here i only used the constant, didn't change any previous behaviour